lib/SIL/SILVerifier.cpp

```diff
@@ -4590,6 +4590,9 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
       }
       if (DS != LastSeenScope) {
         DEBUG(llvm::dbgs() << "Broken instruction!\n"; SI.dump());
```

Comment:
Could we add a more helpful message instead of "Broken Instruction"?
Perhaps: "Instruction re-enters a DebugScope that was already closed at -Onone"?

Reply:
The message we print is:
"SIL verification failed: Basic block contains a non-contiguous lexical scope at -Onone: DS == LastSeenScope"

Does that work for you?